lib/membrane/rtp/twcc_receiver.ex

```diff
@@ -150,4 +159,30 @@ defmodule Membrane.RTP.TWCCReceiver do
       }
     }
   end
+
+  # Checks if actions would be valid in the current context, namely, if they don't reference the pad that hasn't been linked yet
+  # If they are valid, they are executed immediately, otherwise they are buffered and executed as soon as the pad in question is added
+  defp send_as_actions_or_buffer(actions, ctx, state) do
+    is_valid? = &if(Map.has_key?(ctx.pads, &1), do: :valid, else: :invalid)
+
+    grouped =
+      Enum.group_by(actions, fn
+        {action, {pad, _event}} when action in [:event, :buffer, :caps] ->
+          is_valid?.(pad)
+
+        {:end_of_stream, pad} ->
+          is_valid?.(pad)
+
+        _otherwise ->
+          :valid
+      end)
+
+    buffer =
+      Map.get(grouped, :invalid, [])
+      |> Enum.reduce(state.actions_buffer, fn {_action, {pad, _args}} = action, buffer ->
```

Comment:
This will crash on `{:end_of_stream, pad} = action`

Reply:
This is no longer relevant